core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -201,94 +199,101 @@ class RestAPIVersion(config: WhiskConfig, apiPath: String, apiVersion: String)(
-        implicit override val entityStore: EntityStore,
-        override val entitlementProvider: EntitlementProvider,
-        override val activationIdFactory: ActivationIdGenerator,
-        override val loadBalancer: LoadBalancerService,
-        override val executionContext: ExecutionContext,
-        override val logging: Logging,
-        override val whiskConfig: WhiskConfig)
-    extends WhiskPackagesApi with WhiskServices
+            implicit override val entityStore: EntityStore,
+            override val entitlementProvider: EntitlementProvider,
+            override val activationIdFactory: ActivationIdGenerator,
+            override val loadBalancer: LoadBalancerService,
+            override val executionContext: ExecutionContext,
+            override val logging: Logging,
+            override val whiskConfig: WhiskConfig)
+        extends WhiskPackagesApi with WhiskServices {
+        override def changeCacheCallback(key: CacheKey): Future[Unit] = sendCacheInvalidation(key: CacheKey)
+    }
 
     class RulesApi(
         val apiPath: String,
         val apiVersion: String)(
-        implicit override val actorSystem: ActorSystem,
-        override val entityStore: EntityStore,
-        override val entitlementProvider: EntitlementProvider,
-        override val activationIdFactory: ActivationIdGenerator,
-        override val loadBalancer: LoadBalancerService,
-        override val executionContext: ExecutionContext,
-        override val logging: Logging,
-        override val whiskConfig: WhiskConfig)
-    extends WhiskRulesApi with WhiskServices
+            implicit override val actorSystem: ActorSystem,
+            override val entityStore: EntityStore,
+            override val entitlementProvider: EntitlementProvider,
+            override val activationIdFactory: ActivationIdGenerator,
+            override val loadBalancer: LoadBalancerService,
+            override val executionContext: ExecutionContext,
+            override val logging: Logging,
+            override val whiskConfig: WhiskConfig)
+        extends WhiskRulesApi with WhiskServices {
+        override def changeCacheCallback(key: CacheKey): Future[Unit] = sendCacheInvalidation(key: CacheKey)
+    }
 
     class TriggersApi(
         val apiPath: String,
         val apiVersion: String)(
-        implicit override val actorSystem: ActorSystem,
-        implicit override val entityStore: EntityStore,
-        override val entitlementProvider: EntitlementProvider,
-        override val activationStore: ActivationStore,
-        override val activationIdFactory: ActivationIdGenerator,
-        override val loadBalancer: LoadBalancerService,
-        override val executionContext: ExecutionContext,
-        override val logging: Logging,
-        override val whiskConfig: WhiskConfig,
-        override val materializer: ActorMaterializer)
-    extends WhiskTriggersApi with WhiskServices
+            implicit override val actorSystem: ActorSystem,
```

Comment:
can we drop these formatting changes from this pr?
hopefully we will fix formatting wars soon once and for all.